dashboard/lib/projects_list.rb

```diff
@@ -212,14 +212,13 @@ def project_and_featured_project_and_user_fields
     end
 
     def fetch_featured_projects_by_type(project_type)
-      storage_apps = "#{CDO.pegasus_db_name}__storage_apps".to_sym
-
+      storage_apps_table = DCDO.get('storage_apps_in_dashboard', false) ? "#{CDO.dashboard_db_name}__projects".to_sym : "#{CDO.pegasus_db_name}__#{table_name}".to_sym
```

Comment:
Do we need to qualify `table_name` with `StorageApps` here?  Or maybe just hardcode it in?

Reply:
Whoops! This was an artifact of not being able to decide what to do. I'll hard-code it

Reply:
As a side-note, this shows that we do not have very good test coverage around this, since no tests failed 😬